lib/src/resources/subscription.dart

```diff
@@ -32,40 +31,17 @@ class SubscriptionResource extends Resource<Subscription> {
 
   Future<SubscriptionExpanded> retrieveExpanded(
     String id, {
-    required Set<SubscriptionExpandableField> expand,
+    required SubscriptionExpandableField expand,
   }) async {
-    final expandableFields = _expandableFields(expand);
     final response = await get(
       '$_resourceName/$id',
       queryParameters: {
-        'expand': expandableFields.map((e) => e.field).toList(),
+        for (int i = 0; i < expand.innerNestedFields.length; i++)
+          'expand[$i]': expand.innerNestedFields.elementAt(i),
```

Comment:
This will need to be extracted into a reusable piece. Maybe a utility on ExpandableField.